### PR TITLE
Bulk risk acceptance filter remove need for Verified status

### PR DESCRIPTION
**Pull request by madchap**
Addresses #4340

### CODE REVIEW

**Comment by madchap** [comment]
fyi @jvz

File: dojo/models.py [hunk 1]
```diff
@@ -481,7 +481,7 @@ def calc_health(self):
     # only used by bulk risk acceptance api
     @property
     def unaccepted_open_findings(self):
-        return Finding.objects.filter(risk_accepted=False, active=True, verified=True, duplicate=False, test__engagement__product__prod_type=self)
+        return Finding.objects.filter(risk_accepted=False, active=True, verified=False, duplicate=False, test__engagement__product__prod_type=self)
```
Comment (valentijnscholten) [inline dojo/models.py:484]: now you require the findings to be NOT verified. so it will skip verified findings :-)

**Review by valentijnscholten — DISMISSED** [review]

File: dojo/models.py (hunk 1 above)
Comment (madchap) [reply dojo/models.py]: good eye :facepalm:

**Review by valentijnscholten — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Pull request merged**